dev/README.md

```diff
@@ -139,171 +144,31 @@ this command after modifiying the paths specified in the JSON5 file. Four `bigme
-```
-$ spark-submit \
-    --master=spark://$(hostname):7077 \
-    $(which dsgrid-cli.py) \
-    registry \
-    --offline \
-    datasets \
-    register \
-    comstock-derived-dataset/dataset.json5 \
-    query-output/comstock_projected_conus_2022 \
-    -l "Register comstock_projected_conus_2022"
-$ spark-submit \
-    --master=spark://$(hostname):7077 \
-    $(which dsgrid-cli.py) \
-    registry \
-    --offline \
-    projects \
-    submit-dataset \
-    -p dsgrid_conus_2022 \
-    -d comstock_projected_conus_2022 \
-    -m comstock-derived-dataset/dimension_mapping_references.json5 \
-    -l "Submit comstock_projected_conus_2022"
-
-$ spark-submit \
-    --master=spark://$(hostname):7077 \
-    $(which dsgrid-cli.py) \
-    registry \
-    --offline \
-    datasets \
-    register \
-    resstock-derived-dataset/dataset.json5 \
-    query-output/resstock_projected_conus_2022 \
-    -l "Register resstock_projected_conus_2022"
-$ spark-submit \
-    --master=spark://$(hostname):7077 \
-    $(which dsgrid-cli.py) \
-    registry \
-    --offline \
-    projects \
-    submit-dataset \
-    -p dsgrid_conus_2022 \
-    -d resstock_projected_conus_2022 \
-    -m resstock-derived-dataset/dimension_mapping_references.json5 \
-    -l "Submit resstock_projected_conus_2022"
-
-$ spark-submit \
-    --master=spark://$(hostname):7077 \
-    $(which dsgrid-cli.py) \
-    registry \
-    --offline \
-    datasets \
-    register \
-    tempo-derived-dataset/dataset.json5 \
-    query-output/tempo_conus_2022_mapped \
-    -l "Register tempo_conus_2022_mapped"
-$ spark-submit \
-    --master=spark://$(hostname):7077 \
-    $(which dsgrid-cli.py) \
-    registry \
-    --offline \
-    projects \
-    submit-dataset \
-    -p dsgrid_conus_2022 \
-    -d tempo_conus_2022_mapped \
-    -m tempo-derived-dataset/dimension_mapping_references.json5 \
-    -l "Submit tempo_conus_2022_mapped"
-```
-
-4. Dump the registry database to text files.
-```
-singularity run \
-    -B /scratch:/scratch \
-    /projects/dsgrid/containers/arangodb.sif \
-    arangodump \
-    --server.database standard-scenarios \
-    --server.password openSesame \
-    --output-directory simple-standard-scenarios-dump \
-    --compress-output false \
-    --include-system-collections true
-```
-
-5. Copy (e.g., cp, scp, rsync) or arangodump the output files to a dsgrid-test-data
-repository in the directory
-`dsgrid-test-data/filtered_registries/simple_standard_scenarios/dump`, push to GitHub and open a
-pull request.
+3. Copy (e.g., cp, scp, rsync) or arangodump the output files (registry JSON and Parquet) from the
+previous step to a dsgrid-test-data repository in the directory
+`dsgrid-test-data/filtered_registries/simple_standard_scenarios`. Edit the file
+`dump/data_path_*.data.json` to ensure that the `data_path` value is set to
+`dsgrid-test-data/filtered_registries/simple_standard_scenarios`. Make a branch, commit, push
+it to GitHub, and open a pull request.
 
 ### Run tests
```

Comment:
To run tests on Eagle, I assume one needs to get a compute node. Is it also required/advisable to set up a spark cluster first? Is one node for one hour sufficient? Is any ArangoDB set up required? Is it possible to browse the test registry after the tests have run? If so, how?